frontend/packages/topology/src/components/workload/resource-alert.tsx

```diff
@@ -11,13 +11,15 @@ import {
   StatefulSetModel,
 } from '@console/internal/models';
 import { modelFor, referenceFor, referenceForModel } from '@console/internal/module/k8s';
+import { ServiceModel as KnativeServiceModel } from '@console/knative-plugin';
 import { getResource } from '../../utils';
 
 const addHealthChecksRefs = [
   referenceForModel(DeploymentConfigModel),
   referenceForModel(DeploymentModel),
   referenceForModel(DaemonSetModel),
   referenceForModel(StatefulSetModel),
+  referenceForModel(KnativeServiceModel),
```

Comment:
Earlier we were not showing the health check alert on KnativeService. Do we need to now?

Reply:
These are the kebab menu actions for Ksvc
```
if (referenceForModel(resourceKind) === referenceForModel(ServiceModel)) {
      menuActions.push(
        setTrafficDistribution,
        AddHealthChecks,
        EditKsvc,
        EditHealthChecks,
        EditResourceLimits,
      );
    }